### ISSUE TITLE
Doc-strings for methods

### ISSUE THREAD

**Issue by hauleth** [murarth/ketos#17]
It would be cool to allow `define`s to be written like that:

```scheme
(define (foo a b c)
  "Documentation string"
  (+ a b c))
```

This would be great step into making Ketos fully usable language for real development (no need for documentation generator yet).

**Comment by murarth** [comment]
Personally, I prefer a more Rust-like doc comment form:

```lisp
;; Documentation
(define (foo) ...)
```

The tokenizer detects these comments (Vim syntax highlighting, too), but I haven't gotten around to writing a `ketosdoc` utility.

**Comment by hauleth** [comment]
That is somewhat an idea, but docstrings in code has one advantage in Lisps - documentation is part of the code. Which mean you can generate docstrings with macros or work with it like any other piece of language. Also notation with docstrings is de facto standard in Lisp world (AFAIK).

**Comment by murarth** [comment]
That's a good point. Can't argue with that. The doc comment syntax can be sugar for docstrings, anyway. I'll get started implementing this pretty soon.

**Comment by murarth** [comment]
Docstrings have been implemented in afe930caed38e47f53cdf4b5f48acff3879cf7ea.
